ui/articles-aem-renderer/src/androidTest/java/org/cru/godtools/articles/aem/db/AEMDownloadMangerTest.java

```diff
@@ -12,6 +13,7 @@
 public class AEMDownloadMangerTest extends DBBaseTest {
 
     @Test
+    @Ignore
```

Comment:
Is there a reason you are ignoring this test.

Reply:
It was causing Travis to fail due to the test making a network request